### DISCUSSION TITLE
Project sorting

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#765] [Ideas]
Hi 
I would suggest to sort projects on the home page case INsensitive. Currently when number of ones is high it is pretty tough to get around them. Maybe you can implement it ?
Paul

**Comment by cuu508** [comment]
Thanks for the suggestion, that makes sense to me. There are a few other places where we display a list of projects, and it would be nice to use consistent sorting everywhere. Filed issue: #768
